src/pages/home/report/ReportActionsView.js

```diff
@@ -347,8 +338,8 @@ class ReportActionsView extends React.Component {
                 reportID={this.props.reportID}
                 action={item.action}
                 displayAsGroup={this.isConsecutiveActionMadeByPreviousActor(index)}
-                shouldDisplayNewIndicator={this.initialNewMarkerPosition > 0
-                    && item.action.sequenceNumber === this.initialNewMarkerPosition}
+                shouldDisplayNewIndicator={this.props.report.newMarkerSequenceNumber > 0
```

Comment:
I think it would make this code a little cleaner to define this as a `const` at the top of the function.